support-modules/acquisition-events/src/main/scala/com/gu/support/acquisitions/eventbridge/AcquisitionsEventBusService.scala

```diff
@@ -0,0 +1,76 @@
+package com.gu.support.acquisitions.eventbridge
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.eventbridge.model.{PutEventsRequest, PutEventsRequestEntry}
+import com.amazonaws.services.eventbridge.{AmazonEventBridge, AmazonEventBridgeClient}
+import com.gu.aws.CredentialsProvider
+import com.gu.monitoring.SafeLogger
+import com.gu.monitoring.SafeLogger.Sanitizer
+import com.gu.support.acquisitions.models.AcquisitionDataRow
+import com.gu.support.config.Stage
+import com.gu.support.config.Stages.CODE
+import io.circe.syntax.EncoderOps
+import software.amazon.awssdk.core.exception.SdkException
+
+import java.time.Instant
+import java.util.Date
+import scala.concurrent.Promise
+import scala.jdk.CollectionConverters.SeqHasAsJava
+
+class AcquisitionsEventBusService(source: String, stage: Stage, client: AmazonEventBridge) {
+  val eventBusName = s"acquisitions-bus-${stage.toString}"
+  val detailType = "AcquisitionsEvent"
+  def putAcquisitionEvent(acquisition: AcquisitionDataRow) = {
+    val acquisitionJson = acquisition.asJson
+    SafeLogger.info(s"Attempting to send event ${acquisitionJson.spaces2}")
+    val entry = new PutEventsRequestEntry
+    entry.setEventBusName(eventBusName)
+    entry.setSource(source)
+    entry.setDetailType(detailType)
+    entry.setDetail(acquisitionJson.noSpaces)
+    entry.setTime(Date.from(Instant.now))
+
+    val putEventsRequest = new PutEventsRequest
+    putEventsRequest.setEntries(List(entry).asJava)
+
+    val promise = Promise[Either[String, Unit]]
```

Comment:
promis is a type?  I'm not sure how to read this code - is the try on the next line a parameter to the Promise.apply?